e-store/e-store/ScenesScreen/CategoryItems/Cell/ItemsCollectionViewCell.xib

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
add trailing on item image 
remove the constraint between item on the same stack view